packages/back-end/src/scim/users/getUser.ts

```diff
@@ -0,0 +1,72 @@
+import { Response } from "express";
+import { ScimGetRequest } from "../../../types/scim";
+import { getUserById } from "../../services/users";
+
+export async function getUser(req: ScimGetRequest, res: Response) {
+  const userId = req.params.id;
+  console.log("getUser called", userId);
+
+  const user = await getUserById(userId);
+
+  if (!user) {
+    return res.status(404).json({
+      schemas: ["urn:ietf:params:scim:api:messages:2.0:Error"],
```

Comment:
Making a comment for myself - Entra complains when this schema type doesn't include a status property. E.G. 

```
    return res.status(404).json({
      schemas: ["urn:ietf:params:scim:api:messages:2.0:Error"],
      status: "404",
      detail: "User ID does not exist",
    });
    ```